data/themes/classic/style.css

```diff
@@ -618,6 +618,9 @@ TrackContentObjectView   {
 	qproperty-textColor: rgb( 255, 255, 255 );
 	qproperty-textShadowColor: rgb( 0, 0, 0 );
 	qproperty-gradient: true; /* boolean property, set true to have a gradient */
+	/* finger tip offset of cursor */
+	qproperty-mouseHotspotX: 3;
```

Comment:
Can we name this something more specific, since we have several different cursors? `cursorHotspotX-hand`?

For example, when the Knife tool is merged that will have it's own cursor which should ideally be themeable as well. If this is merged as-is and then renamed, we'd end up breaking themes between 1.2 and 1.3 unnecessarily.

Reply:
Wouldn't that knife symbol need to be treated the same way in order to hit the right B&B blocks? With your proposition, we'd then name it `cursorHotspotX-knife`?

Reply:
Yes, that's what I'm thinking.

Reply:
The exact naming scheme is up to you of course, the essence of the suggestion is just to add the specific cursor type somewhere in the property name.

Reply:
Oh I think I get what you mean. We now have only one cursor (per theme), so one "qproperty" is always enough to describe the offset. But if we have two cursors (per theme), we need to qproperties, because the offsets can differ (between knife and hand). Do you mean that?

Reply:
Yes!

Reply:
What if we make a new class which contains all the cursor data?

Also, I'm looking at how to make `QPoint` properties which will allow us to do something like `qproperty-hotspot-hand: 3 3;`

Reply:
@Veratil I like the `QPoint` idea. About the new class, sounds good, but how would our `TrackContentObjectView` access that class? Do you plan that each class that needs these cursor data should inherit that special class? If such a class must inherit `QObject`, we must be careful with diamond inheritance.

Reply:
This PR looks good to merge to me. Do you intend the proposed new class to be implemented here? (_this PR, stable-1.2_)

Reply:
@zonkmachine That was the idea.